### ISSUE TITLE
[Suggestion] Change the Steel crafting to use coal instead of liquid coal 

### ISSUE THREAD

**Issue by ffuentesm** [benbenlaw/Casting#2]
This is mostly due to me finding weird the use of "Liquid Coal" in the process. 

I would suggest using the Solidifier to use Steel; by allowing to place a coal item on the Mold slot to get a steel ingot and a Coal block to get a Steel Block. 
![image](https://github.com/user-attachments/assets/cf750185-6ed6-43a5-a02f-702e71dc857a)

I just find the concept of liquid coal a bit weird, and it would look more realistic to have it use the coal in its solid form instead.

**Comment by hron84** [comment]
Steel is considered as a coal + iron alloy in most of the mods. For example, Mekanism is also having a similar "molten coal" concept, altough it uses the enricher and "molten coal" there is a bit special "fluid".

I know it is very far from realistic but I do not see why it wouldn't fit into all other alloys.

**Comment by benbenlaw** [comment]
yeah this won't change to be honest plus the new multiblock stuff makes alloying better
